script/test

```diff
@@ -381,6 +392,9 @@ main()
                 shift
                 do_cert_suite "$@"
                 ;;
+            unit_tests)
```

Comment:
Since we use cert for cert tests, may we should call this `unit`?
Also, could you please update the help message as well?
```suggestion
            unit)
```